dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -105,7 +109,7 @@ public PasswordResetInitResponse requestPasswordReset(BuiltinUser aUser, boolean
     private String createResetUrl(PasswordResetData passwordResetData) {
         // default to localhost
         String finalHostname = "localhost";
-        String configuredHostname = systemConfig.getFqdn();
+        String configuredHostname = settingsService.getValueForKey(SettingsServiceBean.Key.FQDN);
         if (configuredHostname != null) {
```

Comment:
Same as above

Reply:
I'll do the `isNotBlank` check here

Reply:
Above https://github.com/CeON/dataverse/pull/267#discussion_r308727877